examples/newrelic-express-apm/.env.sample

```diff
@@ -1,5 +1,5 @@
 PORT=8000
-NEW_RELIC_APP_NAME=demo-monitoring-app
+NEW_RELIC_APP_NAME=newrelic-express-apm
 NEW_RELIC_LICENSE_KEY=<new_relic_api_key>
```

Comment:
Does this app uses newrelic over OTEL or is it without OTEL? I'm asking because you have 2x <new_relic_api_key> and I'm wondering if it could be a duplicate.

Reply:
you are right I didn't notice it. :) 
I need to ask to @ajimae

Reply:
@ajimae could you please reply to Lam question?

Reply:
@lojzatran you can optionally use it with OTEL. You only have to pass it as the command line argument stated [here](https://docs.commercetools.com/sdk/observability/newrelic#add-the-new-relic-agent-configuration). I guess maybe we need more documentation on this one.

Reply:
yes, we do it's not clear that we could do it with OTEL as well.
Since I have an opened branch for the official documentation I am gonna include it

Reply:
I don't find any span related to OTEL here, could you help me please?
Not sure if it's working, then I need to add something in the official documentation

Reply:
I'm gonna open a separated task for it and the related documentation